custom/enikshay/nikshay_datamigration/management/commands/create_enikshay_cases.py

```diff
@@ -50,17 +51,31 @@ def add_arguments(self, parser):
             type=str,
         )
         parser.add_argument(
-            '--startswith',
-            dest='startswith',
-            default='',
+            '--location-codes',
```

Comment:
nit: some help text explaining how to use this and what it does could be helpful in case anyone besides you ever needs to run the migration.